Report.md

```diff
@@ -1,69 +1,89 @@
-# **Climate Data Project**
+# Climate Data Project
+
 
 ## Table of Contents:
-# 1. [Introduction](#introduction)
-# 2. [Methodology](#methodology)
-# 3. [Analysis and Findings](#analysis-and-findings)
-# 4. [Discussion](#discussion)
-# 5. [Conclusion](#conclusion)
+- [Climate Data Project](#climate-data-project)
+  - [Table of Contents:](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Methodology](#methodology)
+  - [Analysis and Findings](#analysis-and-findings)
+    - [Temperature](#temperature)
+    - [Humidity](#humidity)
+    - [Precipitation](#precipitation)
+  - [Discussion](#discussion)
+  - [Conclusion](#conclusion)
 
 
-# **Introduction:**
+## Introduction
```

Comment:
In Markdown, a single `#` represents the title, which you have at the top (good). Double `##` is a regular section. You don't need to bold section headings.